Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -0,0 +1,331 @@
+import demistomock as demisto
+from CommonServerPython import *
+from typing import Tuple
+from CommonServerUserPython import *
+# IMPORTS
+import requests
+import ipaddress
+import csv
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+INTEGRATION_NAME = 'Recorded Future'
+
+# CONSTANTS
+SOURCE_NAME = 'recordedfuture.masterrisklist'
+BASE_URL = 'https://api.recordedfuture.com/v2/'
+PARAMS = {'output_format': 'csv/splunk'}
+HEADERS = {'X-RF-User-Agent': 'Demisto',
+           'content-type': 'application/json'}
```

Comment:
I think it may make more sense to add these values as instance variables in your client's `__init__` method.